components/02-molecules/modal/_modal.scss

```diff
@@ -0,0 +1,95 @@
+// Base modal styles. Additional classes accommodate for custom app implementations.
+.modal__toggle {
+  cursor: pointer;
+}
+
+.modal__pane {
+  background-color: clr(background-section);
+  color: clr(text);
+  left: 0;
+  opacity: 0;
+  padding: $space;
+  position: fixed;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: opacity 300ms;
+  visibility: hidden;
+  width: 100vw;
+  z-index: 1000;
```

Comment:
Are all the other compound component properties in alphabetical order? I don't think it matters with the upcoming auto-sorter, but just curious if this was a thing.